### DISCUSSION TITLE
How to contribute?

### DISCUSSION THREAD

**Discussion by pesparzaUT** [Eclectic-Sheep/sheeprl#117] [General]
I would like to contribute to the project

**Comment by belerico** [comment]
Hi @pesparzaUT, if you want we have a bunch of issues opened, like implementing DQN algos or A2C. We are now in the wave of optimizing the buffer also. If you want any help on how to start feel free to contact us

**Comment by pesparzaUT** [comment]
Sounds good, do you have a schedule meeting? Some guidelines to get
familiar with the project structure

**Reply by belerico** [reply]
Sure, when and where do you wanna meet?

**Comment by belerico** [comment]
Hey @pesparzaUT, we have opened a new bunch of issues :metal:
